Subscription detached notice when viewing subscription (#4393)
woocommerce/woocommerce-gateway-stripe: changelog.txt

*** Changelog ***

= 9.7.0 - xxxx-xx-xx =

> * Add - Introduces a new notice that might be displayed when editing a subscription that has no payment method attached

Reviewer: Maybe we can simplify this a bit?
```suggestion
* Add - Show a notice when editing a subscription that has no payment method attached
```

*** Changelog ***

= 9.7.0 - xxxx-xx-xx =

> * Add - Show a notice when editing a subscription that has no payment method attached

Reviewer: Non-blocking: we may want to specify that we only look at active subscriptions.
```suggestion
* Add - Show a notice when editing an active subscription that has no payment method attached
```
